### PR TITLE
Create Terraform template files

### PR DESCRIPTION
**Pull request by ChrisScottThomas**
Create template files for creating Terraform repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+## Usage
+
+### First run
+`pip3 install -r requirements.txt`
+
+### Bootstrap
+
+Replace any mentions of `example` with the name of your new repository, e.g. `burendo-my-service`
+
+Create your AWS session with the cli, and assume the `Administrator` role. For this I persoanlly use [awsume](https://awsu.me/).
```
Comment (steveburtonBUR) [inline README.md]:
```suggestion
Create your AWS session with the cli, and assume the `Administrator` role. For this I personally use [awsume](https://awsu.me/).
```

File: bootstrap.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        time_fetcher=lambda: datetime.datetime.now(tzlocal()),
+    )
+    botocore_session = botocore.session.Session()
+    botocore_session._credentials = creds
+    return boto3.Session(botocore_session=botocore_session)
+
+
+if __name__ == "__main__":
+    main()
+
```
Comment (steveburtonBUR) [inline bootstrap.py]:
```suggestion

```

File: locals.tf.j2 [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  account = {
+    "burendo-prod" = "{{terraform.aws_production_acc}}"
+  }
+
+  tags = {
+    Environment = local.environment
+    Company     = "Burendo"
+    Practice    = "Engineering"
+  }
+
```
Comment (steveburtonBUR) [inline locals.tf.j2]:
```suggestion
  }
```

File: terraform.tf.j2 [hunk 4]
```diff
@@ -0,0 +1,26 @@
+terraform {
+  required_version = "1.2.6"
```
Comment (steveburtonBUR) [inline terraform.tf.j2]: We might want to add this to secrets.

Comment (ChrisScottThomas) [reply terraform.tf.j2]: It is... Bah.

File: bootstrap.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        refresh_using=fetcher.fetch_credentials,
+        time_fetcher=lambda: datetime.datetime.now(tzlocal()),
+    )
+    botocore_session = botocore.session.Session()
+    botocore_session._credentials = creds
+    return boto3.Session(botocore_session=botocore_session)
+
+
+if __name__ == "__main__":
+    main()
```
Comment (steveburtonBUR) [inline bootstrap.py]: Needs a new line

Comment (steveburtonBUR) [reply bootstrap.py]: Not really sure what's happening here, it was a non-empty new line before which was the issue, now it doesn't have one

Comment (ChrisScottThomas) [reply bootstrap.py]: I committed your change. 🤣

**Review by steveburtonBUR — APPROVED** [review]

**Pull request merged**